src/components/TimelineComponent.tsx

```diff
@@ -520,14 +532,15 @@ export default class TimelineComponent
       case ViewType.occurrence:
         this.svg.append('text')
             .attr('transform',
-                `translate(${m.width / 2},${m.height + m.marginTop + 20})`)
+                `translate(${(m.width / 2).toFixed(4)},
+                ${(m.height + m.marginTop + 20).toFixed(4)})`)
             .style('text-anchor', 'start')
             .text(this.state.xColumn);
 
         this.svg.append('text')
             .attr('transform', 'rotate(-90)')
-            .attr('y', 0 - m.marginLeft)
-            .attr('x', 0 - (m.height / 2))
+            .attr('y', (0 - m.marginLeft).toFixed(4))
```

Comment:
Can you clarify why you fixed these numbers?

Reply:
this was done because the snapshot tests were failing because of incredibly small floating point differences that only occurred when testing on sonarcloud.